apps/src/applab/PhoneFrame.jsx

```diff
@@ -0,0 +1,69 @@
+import Radium from 'radium';
+import commonStyles from '../commonStyles';
+import color from '../color';
+import applabConstants from './constants';
+import experiments from '../experiments';
+
+const RADIUS = 30;
+
+const styles = {
+  phoneFrame: {
+    display: 'block',
+    height: 50,
+    backgroundColor: color.lighter_gray,
+  },
+  phoneFrameRunning: {
+    backgroundColor: color.charcoal
+  },
+  phoneFrameTop: {
+    borderTopLeftRadius: RADIUS,
+    borderTopRightRadius: RADIUS,
+  },
+  phoneFrameBottom: {
+    borderBottomLeftRadius: RADIUS,
+    borderBottomRightRadius: RADIUS,
+  },
+  nonResponsive: {
+    maxWidth: applabConstants.APP_WIDTH,
+  }
+};
+
+
+const PhoneFrame = React.createClass({
+  propTypes: {
+    showFrame: React.PropTypes.bool.isRequired,
+    isRunning: React.PropTypes.bool.isRequired
+  },
+
+  render: function () {
```

Comment:
This could easily be a stateless functional component, which leads me to an ES6 question:

If I build a stateless functional component but want to give it `propTypes`, can I still declare the function `const` or do I need to use `let`?  Or do neither...

```
export default function PhoneFrame(props) {
  return ...;
}
PhoneFrame.propTypes = {
  //...
};
```